docs/containers.conf.5.md

```diff
@@ -0,0 +1,339 @@
+ `b` (bytes), `k` (kilobytes), `m`(megabytes), or `g` (gigabytes). 
+ If you omit the unit, the system uses bytes. If you omit the size entirely, 
+the system uses `64m`.
+
+**utsns**="private"
+  Default way to to create a UTS namespace for the container
+  Options are:
+    `private` Create private UTS Namespace for the container.
+    `host`    Share host UTS Namespace with the container.
+
+**userns**="host"
+  Default way to to create a USER namespace for the container
+  Options are:
+    `private` Create private USER Namespace for the container.
+    `host`    Share host USER Namespace with the container.
+
+**userns_size**=65536
+  Number of UIDs to allocate for the automatic container creation. UIDs are
+allocated from the “container” UIDs listed in /etc/subuid & /etc/subgid
+
+## NETWORK TABLE
+The `network` table containers settings pertaining to the management of CNI
+plugins.
+
+**cni_plugin_dirs**=["/opt/cni/bin/",]
+  List of paths to directories where CNI plugin binaries are located.
+
+**default_network**="podman"
+  The network name of the default CNI network to attach pods to.
+
+**network_config_dir**="/etc/cni/net.d/"
+  Path to the directory where CNI configuration files are located.
+
+## LIBPOD TABLE
+The`libpod` table contains configuration options used to set up a libpod runtime
+
+**conmon_env_vars**=""
+  Environment variables to pass into Conmon
+
+**conmon_path**=""
+  Paths to search for the conmon container manager binary. If the paths are 
+empty or no valid path was found, then the `$PATH` environment variable will be
+used as the fallback.
+
+  The default list is
+```
+conmon_path=[
+		"/usr/libexec/podman/conmon",
+		"/usr/local/libexec/podman/conmon",
+		"/usr/local/lib/podman/conmon",
+		"/usr/bin/conmon",
+		"/usr/sbin/conmon",
+		"/usr/local/bin/conmon",
+		"/usr/local/sbin/conmon",
+		"/run/current-system/sw/bin/conmon",
+]
+```
+
+**detach_keys**=""
+  Keys sequence used for detaching a container
+
+**enable_port_reservation**=true
+  Determines whether libpod will reserve ports on the host when they are 
+forwarded to containers. When enabled, when ports are forwarded to containers,
+they are held open by conmon as long as the container is running, ensuring that
+they cannot be reused by other programs on the host. However, this can cause 
+significant memory usage if a container has many ports forwarded to it. 
+Disabling this can save memory.
+
+**events_logger**=""
+  Default method to use when logging events. Valid values:
+ "file", "journald", and "none".
+
+**image_default_transport**="docker://"
+  Default transport method for pulling and pushing images
+
+**infra_command**="/pause"
+  Command to run the infra container
+
+**infra_image** = "k8s.gcr.io/pause:3.1"
+  Infra (pause) container image name for pod infra containers.  When running a 
+pod, we start a `pause` process in a container to hold open the namespaces 
+associated with the  pod.  This container and process, basically sleep/pause
```

Comment:
"container _and_ process" seems strange. Maybe rephrase to "The container does nothing more than to sleep to reserve the pod's resources."?

Reply:
Still open

Reply:
Fixed